src/elements/content-sidebar/SidebarNavSign.tsx

```diff
@@ -1,66 +1,64 @@
-
-export function SidebarNavSign({ blockedReason, intl, status, targetingApi, ...rest }: Props) {
-    const isSignDisabled = !!blockedReason;
-    const isTargeted = targetingApi && targetingApi.canShow;
-    const FtuxTooltip = !isSignDisabled && isTargeted ? TargetedClickThroughGuideTooltip : PlaceholderTooltip;
-    const label = intl.formatMessage(status === 'active' ? messages.boxSignSignature : messages.boxSignRequest);
-    const buttonClassName = classnames('bcs-SidebarNavSign', { 'bdl-is-disabled': isSignDisabled });
-
-    let tooltipMessage = label;
+export type Props = PlainButtonProps;
 
-    switch (blockedReason) {
-        case 'shield-download':
-        case 'shared-link':
-        case 'shield-sign':
-            tooltipMessage = intl.formatMessage(messages.boxSignSecurityBlockedTooltip);
-            break;
-        case 'watermark':
-            tooltipMessage = intl.formatMessage(messages.boxSignWatermarkBlockedTooltip);
-            break;
-        default:
-    }
+export function SidebarNavSign({ ...rest }: Props) {
+    const {
+        blockedReason: boxSignBlockedReason,
+        onClick: onBoxClickRequestSignature,
+        onClickSignMyself: onBoxClickSignMyself,
+        status: boxSignStatus,
+        targetingApi: boxSignTargetingApi,
+        isSignRemoveInterstitialEnabled,
+    } = useFeatureConfig('boxSign');
 
     return (
-        <FtuxTooltip
-            body={intl.formatMessage(messages.boxSignFtuxBody)}
-            position={TooltipPosition.MIDDLE_LEFT}
-            shouldTarget={isTargeted}
-            title={intl.formatMessage(messages.boxSignFtuxTitle)}
-            useTargetingApi={() => targetingApi}
-        >
-            <Tooltip isDisabled={isTargeted} position={TooltipPosition.MIDDLE_LEFT} text={tooltipMessage}>
-                <PlainButton aria-label={label} className={buttonClassName} isDisabled={isSignDisabled} {...rest}>
-                    <BoxSign28 className="bcs-SidebarNavSign-icon" />
-                    <Sign16 width={20} height={20} className="bcs-SidebarNavSign-icon--grayscale" />
-                </PlainButton>
-            </Tooltip>
-        </FtuxTooltip>
+        <>
+            {isSignRemoveInterstitialEnabled ? (
+                <DropdownMenu className="dropdown-menu-sign" isResponsive constrainToWindow isRightAligned>
+                    <SidebarNavSignButton
+                        blockedReason={boxSignBlockedReason}
+                        status={boxSignStatus}
+                        targetingApi={boxSignTargetingApi}
+                        {...rest}
+                    />
+                    <Menu>
+                        <MenuItem
+                            data-testid="sign-request-signature-button"
+                            onClick={() => onBoxClickRequestSignature()}
+                        >
+                            <SignMeOthers32 width={16} height={16} className="sign-menu-icon" />
```

Comment:
classnames should use suit naming: https://github.com/suitcss/suit/blob/master/doc/naming-conventions.md

I think this could be something like `SidebarNavSign-icon`

Reply:
fixed